Jenkinsfile

```diff
@@ -67,7 +67,8 @@ pipeline {
                 script{
                     def data = [
                         "aws_ci": "v1.6.7",
-                        "gce_ci": "v1.7.5"
+                        "gce_ci": "v1.7.5",
+                        "gce_ci_1.8": "v1.8.0"
```

Comment:
could we rename all these so it is obvious in the jenkins ui which version is running for each platform? For example, what about `aws_1.6.7`, `gce_1.7.5`, and `gce_1.8.0`?

Reply:
done, using aws_1.6,gce_1.7 and gce_1.8  for stage names now.